authors/apps/authentication/models.py

```diff
@@ -117,4 +117,25 @@ def get_short_name(self):
         """
         return self.username
 
+    def _generate_jwt_token(self):
+        """
+        Generates the token that stores the user's id
+        It also expires in 5 days
+        """
+        date = datetime.now() + timedelta(days=2)
+        token = jwt.encode({
+            'id' : self.pk,
+            'username' : self.username,
+            'exp' : int(date.strftime('%s'))
+        }, settings.SECRET_KEY, algorithm='HS256')
+
+        return token.decode('utf-8')
+
+    @property
+    def token(self):
+        """
+        This function allows us to get the user token as `user.token` through 
```

Comment:
Update this to show that this generates the token. not `get`

Reply:
I have updated it